docs/src/modules/components/DiamondSponsors.js

```diff
@@ -44,7 +44,7 @@ export default function DiamondSponsors() {
   const t = useTranslate();
 
   return (
-    <Stack direction="column" sx={{ mt: 2 }}>
+    <Stack direction="column" sx={{ mt: 2, mx: 0.5 }}>
```

Comment:
I kill two birds with one stone here. It changes the class name generated, preventing this from working https://github.com/easylist/easylist/blob/18821365f63fa7ee8bcb03f732df2d67fd86dde2/easylist/easylist_specific_hide.txt#L2759 😆

Similar to #44567